README.md

```diff
@@ -252,7 +252,7 @@ A function is often an action on _something_. It is important to state what is i
 
 ```js
 /* A pure function operating with primitives */
-function filter(predicate, list) {
+function filter(list, predicate) {
```

Comment:
I think the value is that it switches the argument, and thus may allow the predicate to be more easily curried (see Ramda’s filter implementation) in the future.

I guess I would say that I slightly prefer the current implementation.